lib/accessories/thermostat.js

```diff
@@ -64,6 +67,10 @@ function FritzThermostatAccessory(platform, ain) {
         .on('get', this.getStatusLowBattery.bind(this))
     ;
 
+    this.services.Thermostat.fritzTargetTemperature = 20;
+    this.services.Thermostat.fritzTargetHeatingCoolingState = Characteristic.TargetHeatingCoolingState.OFF;
+    this.services.Thermostat.fritzCurrentHeatingCoolingState = Characteristic.CurrentHeatingCoolingState.OFF;
```

Comment:
Warum die Defaultwerte? Ist das notwendig?

Reply:
Default Werte sind hier für gewisse Edgecases sinnvoll. Zum einen wenn zum ersten mal der Status der characteristics abgerufen wird, bevor eben diese werte einmal abgerufen wurde. Dann würde das plugin undefined bzw illegale Werte zurück geben. Ist nicht die beste Lösung, weil es vor allem aktuell passieren kann, dass bei einem umschalten auf On die Eingestellen 20 grad die neue TargetTemperature werden können und so die aktuelle TargetTemperature überschrieben wird.
Besser wäre es vllt in den jeweiligen getHandlern auf den call an die API zu warten, sofern die update function noch nicht durchgelaufen ist vor dem ersten Abruf.

Reply:
weiterer Edge case der mit aufgefallen ist, wenn der Thermostat auf aus geschalten ist, wenn das Plugin gestartet wird, und man in HomeKit dann auf HEAT stellt, braucht es irgendeine default temperature